spec/tutorials/discogs_api_tutorial/routing/releases_routing_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe ReleasesController, type: :routing do
+  describe "routing" do
+    it "routes to #index" do
+      expect(get: "/artists/42/releases").to route_to("releases#index", artist_id: '42')
```

Comment:
As we are using this `'42'` string in all routes, can we extract this into a variable?